Analytics/Integrations/Amplitude/SEGAmplitudeIntegration.m

```diff
@@ -29,6 +29,9 @@ - (id)init
 - (void)start
 {
     NSString *apiKey = [self.settings objectForKey:@"apiKey"];
+    if ([(NSNumber *)[self.settings objectForKey:@"trackSessionEvents"] boolValue]) {
+        [Amplitude instance].trackingSessionEvents = true;
```

Comment:
Yes is it ok. In fact, the initialization can send a start session event, so that boolean needs to be set first.

https://github.com/amplitude/Amplitude-iOS#tracking-sessions